src/ctelnet.cpp

```diff
@@ -292,7 +292,7 @@ void cTelnet::connectIt(const QString& address, int port)
 
     hostName = address;
     hostPort = port;
-    QString server = "[ INFO ]  - Looking up the IP address of server:" + address + ":" + QString::number(port) + " ...";
+    QString server = tr("[ INFO ]  - Looking up the IP address of server:") + address + ":" + QString::number(port) + " ...";
```

Comment:
The `QString::operator+` is the same as `QString::append()` {under most circumstances, though there is a workaround that swaps it for `QStringBuilder::operator%` IIRC which works more efficiently}. It would have been better to merge all the concatenations into a single instantiation using the `QString:arg()` method to drop in `address` and `port` replaceable arguments...

Reply:
As above noted as: https://github.com/Mudlet/Mudlet/issues/2123